signac/contrib/project.py

```diff
@@ -1506,7 +1506,7 @@ def clone(self, job, copytree=syncutil.copytree):
         job : :class:`~signac.contrib.job.Job`
             The job to copy into this project.
         copytree :
-             (Default value = syncutil.copytree)
+             (Default value = shutil.copytree)
```

Comment:
This will appear with a link in the documentation.
```suggestion
             (Default value = :func:`shutil.copytree`)
```